_example/simple/simple.go

```diff
@@ -35,20 +41,30 @@ func main() {
 	if err != nil {
 		log.Fatal(err)
 	}
-	defer stmt.Close()
+	defer func() {
+		if err := stmt.Close(); err != nil {
+			log.Fatal(err)
+		}
+	}()
 	for i := 0; i < 100; i++ {
 		_, err = stmt.Exec(i, fmt.Sprintf("こんにちわ世界%03d", i))
 		if err != nil {
 			log.Fatal(err)
 		}
 	}
-	tx.Commit()
+	if err := tx.Commit(); err != nil {
```

Comment:
You should defer `tx.Rollback()` right after `db.Begin()` above for completeness. Only log if it returns an error other than `sql.ErrTxDone`.